accumulo/src/main/java/com/yahoo/ycsb/db/accumulo/AccumuloClient.java

```diff
@@ -100,62 +104,72 @@ public void init() throws DBException {
   @Override
   public void cleanup() throws DBException {
     try {
-      if (bw != null) {
-        bw.close();
+      Iterator<BatchWriter> iterator = writers.values().iterator();
+      while (iterator.hasNext()) {
+        BatchWriter writer = iterator.next();
+        writer.close();
```

Comment:
At the cleanup point, there isn't much else we can do but throw an exception. The cleanup here is best-effort, but probably unnecessary (I don't have a use-case for how YCSB would reasonably recover from this state)